src/main/java/org/opentripplanner/util/PolylineEncoder.java

```diff
@@ -13,53 +10,22 @@
 
 public class PolylineEncoder {
 
-  public static EncodedPolylineBean createEncodings(double[] lat, double[] lon) {
-    return createEncodings(new PointAdapterList(lat, lon));
-  }
-
-  public static EncodedPolylineBean createEncodings(double[] lat, double[] lon, int level) {
-    return createEncodings(new PointAdapterList(lat, lon), level);
-  }
-
-  public static EncodedPolylineBean createEncodings(
-    double[] lat,
-    double[] lon,
-    int offset,
-    int length,
-    int level
-  ) {
-    return createEncodings(new PointAdapterList(lat, lon, offset, length), level);
-  }
-
-  public static EncodedPolylineBean createEncodings(Iterable<Coordinate> points) {
-    return createEncodings(points, -1);
-  }
-
   public static EncodedPolylineBean createEncodings(Geometry geometry) {
```

Comment:
These methods are called `createEncodings` but they don't create encodings, aren't they? Can we rename them?

Reply:
done